project/.php_cs.dist

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * DO NOT EDIT THIS FILE!
+ *
+ * It's auto-generated by sonata-project/dev-kit package.
+ *
+ * Package `sllh/php-cs-fixer-styleci-bridge` is required to get it working.
```

Comment:
This is still true?

Reply:
It's just a bad copy paste from the old file.